src/core/frames/frame_controller.ts

```diff
@@ -266,12 +266,16 @@ export class FrameController
     const frame = this.findFrameElement(formSubmission.formElement, formSubmission.submitter)
 
     frame.delegate.proposeVisitIfNavigatedWithAction(frame, formSubmission.formElement, formSubmission.submitter)
-
     frame.delegate.loadResponse(response)
+
+    if (formSubmission.method !== FetchMethod.get) {
```

Comment:
For what it's worth, there's a [FormSubmission.isIdempotent](https://github.com/hotwired/turbo/blob/39affe528cd01be9b8cb0cb0e8091997c254f3ed/src/core/drive/form_submission.ts#L113-L115) property with this implementation.

Reply:
Thanks 👍 I saw that, but used the same check as the `Navigator` since the cache logic isn't really about idempotency. But I think actually a better option is to rename that method, as it was a bit misleading. Then we can use it. I've done that in #883.